clash-prelude/src/Clash/Sized/RTree.hs

```diff
@@ -537,3 +544,93 @@ lazyT :: KnownNat d
       => RTree d a
       -> RTree d a
 lazyT = tzipWith (flip const) (trepeat ())
+
+-- | Low-depth (left) scan, see 'Clash.Sized.Vector.scanl1'. The generated
+-- circuit will use more gates but can be clocked faster.
```

Comment:
```suggestion
-- | Low-depth (left) scan, see 'Clash.Sized.Vector.scanl1'. The generated
-- circuit will do O(2*n) operations, but has a logic depth of O(log n);
-- i.e. it will use more gates but can be clocked faster.
```